mate-panel/libmate-panel-applet-private/panel-applet-container.c

```diff
@@ -629,32 +620,33 @@ mate_panel_applet_container_child_set (MatePanelAppletContainer *container,
 {
 	GDBusProxy               *proxy = container->priv->applet_proxy;
 	const AppletPropertyInfo *info;
-	GSimpleAsyncResult       *result;
-
+	GTask                    *task;
 	if (!proxy)
 		return NULL;
 
 	info = mate_panel_applet_container_child_property_get_info (property_name);
 	if (!info) {
-		g_simple_async_report_error_in_idle (G_OBJECT (container),
+		g_task_report_new_error (G_OBJECT (container),
 						     callback, user_data,
+							 NULL,
 						     MATE_PANEL_APPLET_CONTAINER_ERROR,
 						     MATE_PANEL_APPLET_CONTAINER_INVALID_CHILD_PROPERTY,
 						     "%s: Applet has no child property named `%s'",
 						     G_STRLOC, property_name);
```

Comment:
Indents

Reply:
Line 630 - 635 should start after the bracket `(G_OBJECT` of previous line.

Reply:
Line 630,632 - 635  I kept the original indent format and didn't change anything

Reply:
You shortened line `629`, in result you have to aligned all lines of this code part. Or does this looks worse?
```
 	if (!info) {
-		g_simple_async_report_error_in_idle (G_OBJECT (container),
+		g_task_report_new_error (G_OBJECT (container),
 					 callback, user_data,
+					 NULL,
 					 MATE_PANEL_APPLET_CONTAINER_ERROR,
 					 MATE_PANEL_APPLET_CONTAINER_INVALID_CHILD_PROPERTY,
 					 "%s: Applet has no child property named `%s'",
@@ -645,16 +636,17 @@ mate_panel_applet_container_child_set (MatePanelAppletContainer *container,
 		return NULL;
 	}
```